### ISSUE TITLE
question about different models

### ISSUE THREAD

**Issue by lwtai** [Xinjie-Q/GaussianImage#16]
Hi, thanks for the nice and concise works!

the difference between "gaussianimage_rs" and "gaussianimage_cholesky" has mentioned clearly in your paper, and they further discard the needs of view matrix compared to "gaussiansplatting_3d"

I am curious about the implementation "gaussiansplatting_3d", since it didn't complete remove the input of viewpoint, do you try the performance render from slightly difference viewpoint?

**Comment by Xinjie-Q** [comment]
No, I do not try different viewpoints. I use the viewpoint from the single image fitting example of gsplat.

**Comment by lwtai** [comment]
Got it, thanks for replying
I also wonder if your custom 2d gaussian cuda code supports varying channels rendering, for example single channel or more than three channels, like gsplat does?

**Comment by Xinjie-Q** [comment]
Currently the released gaussianimage code does not support varying channels rendering. Since I'm busy looking for a job recently, I don't have the free time to support this feature at the moment. But I will support this feature after I finish looking for a job and have some free time.

**Comment by lwtai** [comment]
I see, thanks! 祝你找工作順利
